src/components/IconToggle/IconToggle.tsx

```diff
@@ -19,22 +19,45 @@ export interface IconToggleProps extends TPAComponentProps {
   onChange?(event: OnChangeEvent): void;
   checked: boolean;
   disabled: boolean;
+  animation: boolean;
```

Comment:
I don't remember if we talked about adding an `animation` property.
Even though it's a great idea to avoid introducing a breaking change,
I think in this case we don't need to add the prop.
The animation can be by default.